go/vt/workflow/resharding/horizontal_resharding_workflow.go

```diff
@@ -76,19 +76,21 @@ func (*HorizontalReshardingWorkflowFactory) Init(m *workflow.Manager, w *workflo
 	subFlags := flag.NewFlagSet(horizontalReshardingFactoryName, flag.ContinueOnError)
 	keyspace := subFlags.String("keyspace", "", "Name of keyspace to perform horizontal resharding")
 	vtworkersStr := subFlags.String("vtworkers", "", "A comma-separated list of vtworker addresses")
+	minHealthyRdonlyTablets := subFlags.String("min_healthy_rdonly_tablets", "", "Minimum number of rdonly healthy tablets required")
```

Comment:
Minor nit -- I'd rephrase to "Minimum number of healthy RDONLY tablets required"